layouts/partials/components/feedback.html

```diff
@@ -0,0 +1,37 @@
+  class="hx-px-6  hx-py-3 hx-rounded-lg hx-border hx-border-gray-100 hx-text-current hx-no-underline dark:hx-shadow-none dark:hx-bg-dark hover:hx-shadow-gray-100 dark:hx-hover:hx-shadow-none hx-shadow-gray-100 active:hx-shadow-sm active:hx-shadow-gray-200 hx-transition-all hx-duration-200 hover:hx-border-gray-300  hx-shadow-sm dark:hx-border-neutral-700 hover:hx-bg-slate-50 hover:hx-shadow-md dark:hover:hx-border-neutral-700 dark:hover:hx-bg-neutral-700"
+>
+  <svg aria-hidden="true" focusable="false" role="img" class="color-fg-muted" viewBox="0 0 16 16" width="16" height="16" fill="currentColor" style="display:inline-block;user-select:none;vertical-align:text-bottom;overflow:visible"><path d="M7.083 15.986c-.763-.087-1.499-.295-2.011-.884-.504-.581-.655-1.378-.655-2.299 0-.468.087-1.12.157-1.638l.015-.112H3.167c-.603 0-1.174-.086-1.669-.334a2.415 2.415 0 0 1-1.136-1.2c-.454-.998-.438-2.447-.188-4.316l.04-.306C.32 4.108.41 3.424.526 2.864c.132-.63.316-1.209.669-1.672C1.947.205 3.211.053 4.917.053c1.848 0 3.234.332 4.388.652l.474.133c.658.187 1.201.341 1.726.415a1.75 1.75 0 0 1 1.662-1.2h1c.966 0 1.75.784 1.75 1.75v7.5a1.75 1.75 0 0 1-1.75 1.75h-1a1.75 1.75 0 0 1-1.514-.872c-.259.105-.59.268-.919.508-.671.491-1.317 1.285-1.317 2.614v.5c0 1.201-.994 2.336-2.334 2.183Zm4.334-13.232c-.706-.089-1.39-.284-2.072-.479l-.441-.125c-1.096-.304-2.335-.597-3.987-.597-1.794 0-2.28.222-2.529.548-.147.193-.275.505-.393 1.07-.105.502-.188 1.124-.295 1.93l-.04.3c-.25 1.882-.19 2.933.067 3.497a.923.923 0 0 0 .443.48c.208.104.52.175.997.175h1.75c.685 0 1.295.577 1.205 1.335-.022.192-.049.39-.075.586-.066.488-.13.97-.13 1.329 0 .808.144 1.15.288 1.316.137.157.401.303 1.048.377.307.035.664-.237.664-.693v-.5c0-1.922.978-3.127 1.932-3.825a5.878 5.878 0 0 1 1.568-.809Zm1.75 6.798h1a.25.25 0 0 0 .25-.25v-7.5a.25.25 0 0 0-.25-.25h-1a.25.25 0 0 0-.25.25v7.5c0 .138.112.25.25.25Z"></path></svg></button>
+</div>
+</p>
+<script>
+  const feedbackSubmitContainer = document.getElementById("feedback-submit-container");
+  const result = document.getElementById("feedback-end");
+  function submitedFeedback(feedbackSentisPositive) {
+    feedbackSubmitContainer.remove();
+    if (feedbackSentisPositive) {
+      result.innerHTML = '{{ .Site.Params.feedback.successMsg | default "Glad to hear it, thank you for the feedback!" }}';
+    } else {
+      result.innerHTML = '{{ .Site.Params.feedback.successMsg | default "Sorry to hear that. Please <a href=\"https://github.com/medic/cht-docs/issues/new\">tell us how we can improve." }}';
+    }
+  }
+</script>
+<!-- Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id={{ site.Params.services.googleAnalytics.id }}"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+  gtag('config', '{{ site.Params.services.googleAnalytics.id }}');
```

Comment:
The `googleAnalytics` value was not pulling through for me when the page actually rendered:

![image](https://github.com/user-attachments/assets/d81620ed-0e65-4fe2-bbf8-df9fb5e3e141)

Checked [the docs](https://gohugo.io/configuration/services/#googleanalyticsid) and it looks like we should reference `.Site.Config.Services.GoogleAnalytics.ID` instead:


```suggestion
<script async src="https://www.googletagmanager.com/gtag/js?id={{ .Site.Config.Services.GoogleAnalytics.ID }}"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', '{{ .Site.Config.Services.GoogleAnalytics.ID }}');
```